resources/validate_builds.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+# validated if created build is debug build 
+cd $APK_PATH
+
+APK_NAME=""
+EXPECTED_FLAG='application-debuggable'
+
+for file in *.apk
+do
+  APK_NAME=$file  
+done
+
+DEBUGGABLE=`$ANDROID_HOME/build-tools/27.0.3/aapt dump badging $APK_NAME |grep $EXPECTED_FLAG`
+if [ $DEBUGGABLE = $EXPECTED_FLAG ]
+   then
+    echo "Build is debuggable"
```

Comment:
This script will run and succeed regardless of whether or not the build is 'debuggable'. Most people are not going to read the logging in the Jenkins build, so they will just trust that a passing build is 'debuggable'. I would suggest something like the following:
```
ANDROID_HOME/build-tools/27.0.3/aapt dump badging $APK_NAME |grep $EXPECTED_FLAG
if [ $? == 0 ]; then
   echo "The build is debuggable"
   exit 0
else
   echo "The build is not debuggable"
   exit 1
fi
```

Reply:
done, thanks.